sdk/core/core-util/vitest.browser.config.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { defineConfig } from "vitest/config";
+
+export default defineConfig({
+  test: {
+    browser: {
```

Comment:
Is playwright based on chromium? Could we include in comments configs to test other browsers (only firefox now I guess)? Do we need additional dependencies to test firefox?

Reply:
[Playwright](https://playwright.dev/) is a generic test runner (sponsored by Microsoft) -- `npx playwright install` handles downloading their supported browsers, which I believe includes Firefox. `vitest` also supports webdriverio as an alternative. I don't have a strong preference, but I figured we could try the MSFT solution first.